src/state/zustand/swapperStore/amountSelectors.ts

```diff
@@ -0,0 +1,394 @@
+import { createSelector } from 'reselect'
+import { TradeAmountInputField } from 'components/Trade/types'
+import { bnOrZero } from 'lib/bignumber/bignumber'
+import { fromBaseUnit, toBaseUnit } from 'lib/math'
+import type { SwapperState } from 'state/zustand/swapperStore/useSwapperStore'
+
+const selectAssetPriceRatio = createSelector(
+  (state: SwapperState) => state.buyAssetFiatRate,
+  (state: SwapperState) => state.sellAssetFiatRate,
+  (buyAssetFiatRate, sellAssetFiatRate) => {
+    return bnOrZero(buyAssetFiatRate).dividedBy(bnOrZero(sellAssetFiatRate)).toString()
```

Comment:
`toFixed()` for a more accurate ratio?

Reply:
https://github.com/shapeshift/web/pull/4138/commits/1a19e737735ec7cad27c0e90d91e0fcc14207446